PURL-SPECIFICATION.rst

```diff
@@ -206,44 +206,78 @@ The rules for each component are:
     in the canonical form
   - Each ``subpath`` segment MUST be a percent-encoded string
   - When percent-decoded, a segment:
+
     - MUST NOT contain a '/'
     - MUST NOT be any of '..' or '.'
     - MUST NOT be empty
+
   - The ``subpath`` MUST be interpreted as relative to the root of the package
 
 
 Character encoding
 ~~~~~~~~~~~~~~~~~~
 
-For clarity and simplicity a ``purl`` is always an ASCII string. To ensure that
-there is no ambiguity when parsing a ``purl``, separator characters and non-ASCII
-characters must be UTF-encoded and then percent-encoded as defined at::
+A canonical ``purl`` is always an ASCII string composed only of these characters:
+
+- ``A to Z``,
+- ``a to z``,
+- ``0 to 9`` and
+- the punctuation marks ``:/@?#%.-_~`` .
```

Comment:
what about `+` (plus)

we are not www-URL, so a plus is a plus. while a space, which is not in the list of allowed characters, needs to be percent-encoded .... which is declared already. but maybe add also a explicit note how spaces should be handled

Reply:
#261 recommends that a PURL qualifier value should not contain `+` because some implementations do treat it as ` `. But, in the interest of simplifying the encoding, maybe the specified encoding should be the same for qualifier values as everywhere else (whether that's encoded or not).

Reply:
these implementations are already not respecting the current purl spec.